src/chttpd/src/chttpd_misc.erl

```diff
@@ -68,15 +68,10 @@ handle_welcome_req(Req, _) ->
     send_method_not_allowed(Req, "GET,HEAD").
 
 get_features() ->
-    case dreyfus:available() of
-        true -> [search];
-        false -> []
-    end ++
-        case nouveau:enabled() of
-            true -> [nouveau];
-            false -> []
-        end ++
-        config:features().
+    HasSearch = [search || dreyfus:available()],
+    HasNouveau = [nouveau || nouveau:enabled()],
+    HasQuickJS = [quickjs || couch_server:get_js_engine() == <<"quickjs">>],
```

Comment:
a small preference for `/= <<"spidermonkey">>` since that's the one I think you are striving not to add (otherwise we'd just add the result of get_js_engine() in all cases, which arguably is more helpful anyway)

Reply:
oh great point, I agree, will update it